src/modules/geocoder/use-geocoder.ts

```diff
@@ -37,9 +38,10 @@ export function useGeocoder(
             locations: response?.data?.map(mapFeatureToLocation),
           });
         } catch (err) {
-          if (!isCancel(err)) {
+          const error = err as ErrorResponse;
+          if (!(error.kind === 'AXIOS_CANCEL')) {
             console.warn(err);
-            dispatch({type: 'SET_ERROR', error: getAxiosErrorType(err)});
+            dispatch({type: 'SET_ERROR', error: error.kind as AxiosErrorKind});
```

Comment:
Is casting to `AxiosErrorKind` safe? `error.kind` can at least in theory have other values than the ones listen in `AxiosErrorKind`.

Reply:
It is probably not safe, as if casting ever is 😅 Unsure how we should handle this though.